tests/test_trainer.py

```diff
@@ -492,6 +493,15 @@ def test_mixed_bf16(self):
 
         # will add more specific tests once there are some bugs to fix
 
+    @require_torch_gpu
+    @require_torch_tf32
```

Comment:
Out of curiosity, do we have a setup that has the right CUDA version an GPU capabilities?

Reply:
I have rtx-3090 if that's what you ask.

Running benchmarks now - will post those shortly.

Reply:
I was wondering for our testing machines on the automatic CI :-)

Reply:
one day we will have those newer gpus.